src/utils.py

```diff
@@ -129,23 +137,25 @@ def submit_updates(
         else:
             raise
 
-    # create a new commit updating the aggregate flow metadata file
-    updated_metadata_content = json.dumps(updated_metadata_dict, indent=2)
-    if existing_metadata_content == updated_metadata_content:
-        print(
-            f"Aggregate {variety} metadata for {collection_name} {latest_release} already up to date!"
-        )
-        return
+    # don't update the aggregate metadata with keys that have empty values
+    if collection_metadata:
+        # create a new commit updating the aggregate flow metadata file
+        updated_metadata_content = json.dumps(updated_metadata_dict, indent=2)
+        if existing_metadata_content == updated_metadata_content:
+            print(
+                f"Aggregate {variety} metadata for {collection_name} {latest_release} already up to date!"
+            )
+            return
 
-    registry_repo.file_contents(metadata_file, ref=branch_name).update(
-        message=f"Update aggregate {variety} metadata with `{collection_name}` `{latest_release}`",
-        content=updated_metadata_content.encode("utf-8"),
-        branch=branch_name,
-    )
+        registry_repo.file_contents(metadata_file, ref=branch_name).update(
+            message=f"Update aggregate {variety} metadata with `{collection_name}` `{latest_release}`",
```

Comment:
For consistent terminology, should "aggregate" be "view" instead

Reply:
i dont think so, since "Update view block metadata" doesn't make direct sense to me

maybe you mean "metadata" should be "view"?